<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieGenresAddModifyForm</class>
 <widget class="QDialog" name="MovieGenresAddModifyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>181</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/movie genre pic.png</normaloff>:/Images/movie genre pic.png</iconset>
  </property>
  <widget class="QPushButton" name="pbtOk">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblId">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledGenre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblIdlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
